clients/drcachesim/tools/invariant_checker.cpp

```diff
@@ -378,15 +378,13 @@ invariant_checker_t::parallel_shard_memref(void *shard_data, const memref_t &mem
         memref.instr.type == TRACE_TYPE_PREFETCH_INSTR ||
```

Comment:
Before submitting, please confirm that asan passes with these changes.

Reply:
Yes, I have confirmed that asan passes with these changes.